src/lake_modelling/utils/user_inputs.py

```diff
@@ -0,0 +1,69 @@
+import streamlit as st
+
+
+def get_initial_cond():
+    st.markdown("### Initial Conditions")
+    col1, col2 = st.columns(2)
+    C_lake0 = col1.number_input(
+        "Initial lake Ca concentration (mg-Ca/l)", min_value=0, value=1
+    )
+    C_in0 = col1.number_input(
+        "Lake inflow Ca concentration (mg-Ca/l)", min_value=0, value=1
+    )
+    C_bott0 = col2.number_input(
+        "Lake bottom concentration (mg-Ca/l)", min_value=0, value=0
+    )
+    pH_lake0 = col2.number_input("Lake pH", min_value=1.0, max_value=7.0, value=4.5)
+
+    init_cond = C_lake0, C_in0, C_bott0, pH_lake0
+
+    return init_cond
+
+
+def get_lake_params():
+    st.markdown("### Lake characteristics")
+    col1, col2 = st.columns(2)
+
+    area = col1.number_input("Lake area (km2)", min_value=0.0, value=1.14)
+    mean_depth = col1.number_input("Mean lake depth (m)", min_value=0.0, value=5.6)
+    res_time = col2.number_input("Volume / mean_annual_q", min_value=0, value=10)
+    flow_profile = col2.selectbox("Choose profile", ("none", "fjell", "kyst"))
+
+    lake_params = (area, mean_depth, res_time, flow_profile)
+
+    return lake_params
+
+
+def get_duration():
+    st.markdown("### Model setup")
+    n_months = st.number_input("Number of months to simulate", min_value=0, value=12)
```

Comment:
@JamesSample Is this ok with min=2?

Reply:
I think strictly `min` should be one. Allowing `min=0` does not make sense, as that means you don't want to simulate anything at all, but it should be possible to simulate just the first month, if desired (although in practice we'll almost always want to simulate more than this).

Reply:
I don't understand why this isn't showing the updated code here. `min=0` is outdated code. 

I initially wrote `min_value=2`, since the min liming_month is 1, but then I realized makes more sense to leave the simulation minimum value as 1 and add one to the stored value afterwards `return n_months + 1` (since month count in the simulation starts at 1, not a zero). 

Then if the user wants to simulate one month, the input will be 1, not 2 :smile: way more sensible